subsys/bluetooth/services/fast_pair/fp_storage/include_priv/fp_storage_priv.h

```diff
@@ -23,18 +26,18 @@
 extern "C" {
 #endif
 
-/** Settings subtree name for Fast Pair storage.  */
-#define SETTINGS_AK_SUBTREE_NAME "fp"
+/** Settings subtree name for Fast Pair storage. */
+#define SETTINGS_AK_SUBTREE_NAME "fp_ak"
```

Comment:
It would be good to have all the Fast Pair data in one subtree

fp/ak
fp/pn

and so on...

If the subtree deletion was ever to be implemented in the Settings subsystem, we could use one operation to wipe all Fast Pair (settings_delete("fp")) in case of a factory reset event or other circumstances

Reply:
Marek's idea was to separate Account Key storage from the rest as in future it theoretically may use other more secure storage backend (https://projecttools.nordicsemi.no/jira/browse/NCSDK-15912). For me it doesn't really matter. There is also an issue with extensions and whether they should have separate storage as they can be turned on/off - we decided to have storage for Account Keys (maybe more secure in the future) and storage for personalized name as other extensions won't need storage.

Reply:
With the suggested naming scheme, you still get separation because you can refer to each storage part using the respective path:  fp/ak and fp/pn. However, you also have an option to delete all Fast Pair data in one go by using fp/ path

I also don't see how this can block the secure storage backend for Account Key storage. As I said already, you can still identify all Account Keys with the fp/ak path.

Let's wait for Marek to weigh in.